algebra/benches/fft/mod.rs

```diff
@@ -0,0 +1,121 @@
+use algebra::{
+    fields::mnt6753::Fr,
+    fft::get_best_evaluation_domain,
+    UniformRand,
+};
+use rand::SeedableRng;
+use rand_xorshift::XorShiftRng;
+
+#[bench]
+fn bench_basic_domain_fft(b: &mut ::test::Bencher) {
+    const SAMPLES: usize = 1000;
+    let mut rng = XorShiftRng::seed_from_u64(1231275789u64);
+
+    let num_coeffs = 1 << 14;
```

Comment:
I'd add a comment here, to emphasize that num_coeffs is a power of 2 (and does not exceed the maximum duadic domain size 2^15 of the MNT4 Fq) hence get_best_evaluation_domain sets up a duadic FFT domain.